Bottom: 59354f1d3aa16f538d85d39f7fcedecd299357a3
Top:    b815f7908d4146a972f4724d8a08331e0a251723
Author: Jared Casper <jaredcasper@gmail.com>
Date:   2010-06-05 23:46:39 +0100

Fix pr2717258, Lock mode with F12.

Binds F12 to lock mode in the default gcpb-menu.res and pcb-menu.res
files, bringing them inline with existing documentation.


---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 37e141d..df307be 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -516,7 +516,7 @@ PopupMenus =
        {"Insert Point" checked=insertpointmode,1 Mode(InsertPoint) a={"Insert" "<Key>Insert"}}
        {"Move" checked=movemode,1 Mode(Move)}
        {"Copy" checked=copymode,1 Mode(Copy)}
-       {"Lock" checked=lockmode,1 Mode(Lock)}
+       {"Lock" checked=lockmode,1 Mode(Lock) a={"F12" "<Key>F12"}}
        {"Cancel" Mode(Escape) a={"Esc" "<Key>Escape"}}
       }
     }
diff --git a/src/pcb-menu.res b/src/pcb-menu.res
index cbdcd2d..38e86e4 100644
--- a/src/pcb-menu.res
+++ b/src/pcb-menu.res
@@ -182,7 +182,7 @@ MainMenu =
    {"Insert Point" checked=insertpointmode,1 Mode(InsertPoint) a={"Insert" "<Key>Insert"}}
    {"Move" checked=movemode,1 Mode(Move)}
    {"Copy" checked=copymode,1 Mode(Copy)}
-   {"Lock" checked=lockmode,1 Mode(Lock)}
+   {"Lock" checked=lockmode,1 Mode(Lock) a={"F12" "<Key>F12"}}
    {"Cancel" Mode(Cancel) a={"Esc" "<Key>Escape"}}
    -
    {"Command" Command() a={":" "<Key>:"}}
